rir/src/interpreter/interp_data.h

```diff
@@ -381,11 +383,17 @@ typedef struct Function {
     unsigned markOpt : 1;
     unsigned spare : 28;
 
+    // TODO(mhyee): remove origin and next
```

Comment:
I tried removing `origin` and `next` in this patch, but it's actually really invasive. It would break deoptimization and the [hack](https://github.com/reactorlabs/rir/pull/54/files#diff-ddb2b0e78b44bcab154f459281266a97R333) that prevents optimizing a function twice.

Reply:
uh, ok, too bad